Verificaiton.md

```diff
@@ -0,0 +1,38 @@
+# License Verification Platform Technical Documentation
+The purpose of this document is to provide documentation for concepts relating to the [License Verification Platform](https://verificaiton.clicknclear.com) that do not fall under the API documentation.
```

Comment:
Broken link due to typo

Reply:
ba64c5a

Reply:
Ahhh sorry mis-read: c27a558ce28c886be58ca9143d38b389bd44d264